services/example-exercise/src/components/ExerciseBase.tsx

```diff
@@ -29,21 +31,25 @@ const ExerciseBase: React.FC<Props> = ({
     >
       {alternatives.map((option) => {
         const selected = selectedId === option.id
+        const correct = model_solutions.includes(option.id)
+        const hasSolutions = model_solutions.length > 0
```

Comment:
When we have no solutions pass null instead of an empty array. An empty array should mean that all alternatives are wrong.